### PR TITLE
[READY] add tracking for vimeo videos

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Needs testing and maybe we need to seperate the home from the tour video.

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
It's exactly this: https://github.com/sanderheilbron/vimeo.ga.js

```'progress25' : false,
          'progress50' : false,
          'progress75' : false,
          'videoPlayed' : false,
          'videoPaused' : false,
          'videoResumed' : false,
          'videoSeeking' : false,
          'videoCompleted' : false,
          'timePercentComplete' : 0```

It sends events to Google Analytics: 
![videos](https://cloud.githubusercontent.com/assets/16101007/19269230/c23b3b7c-8fb9-11e6-953c-0bc757bedc0f.png)

**Comment by fatboypunk** [comment]
Can you fix hound?

**Comment by RonaldDijkstra** [comment]
2 violations left, how to fix those?

File: source/javascripts/vimeo.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    // Handle messages received from the player
+    onMessageReceived: function(e) {
+      if (e.origin.replace('https:', 'http:') !== "http://player.vimeo.com" ||
+        typeof vimeoGAJS.gaTracker === 'undefined') {
+        //console.warn('Tracker is missing!');
+        return;
+      }
+
+      var data = JSON.parse(e.data),
+          iframeEl = $('#'+data.player_id),
```
Comment (RonaldDijkstra) [inline source/javascripts/vimeo.js]: This is required, can't fix this, is a Vimeo parameter.

**Comment by RonaldDijkstra** [comment]
Ready!

**Comment by fatboypunk** [comment]
damn, forgot about this, sorry

**Comment by RonaldDijkstra** [comment]
me too! 💃

**Pull request merged**